easybuild/easyconfigs/p/PLINK/PLINK-1.07_redeclaration.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/sets.cpp b/sets.cpp
```

Comment:
please add a comment above this line what this patch is about, where you got it, and who created it (you?)
any text above the `diff` line is comments